### DISCUSSION TITLE
How to configure font fallback so that it supports Emojis?

### DISCUSSION THREAD

**Discussion by AnsonH** [neovide/neovide#1156] [Q&A] [answered]
From the repo README, it says Neovide supports rendering of emoji not contained in the configured font via font fallback. However, I'm not sure how exactly can I achieve that.

I tried the following settings (I'm using Windows):
```
set guifont=FiraCode\ NF:h14.5,Segoe\ UI\ Emoji
```

However, it doesn't look like it's working:
![image](https://user-images.githubusercontent.com/57580593/150067701-8e33f328-7e17-4683-a371-8e81cee6ddc7.png)

**Comment by Kethku** [comment, marked as answer]
put the comma before the :h14.5 along with the font you want to fallback to

**Reply by Kethku** [reply]
`set guifont=FiraCode\ NF,Segoe\ UI\ Emoji:h14.5`

**Reply by AnsonH** [reply]
Sorry it's still not working for some reason (same results as the screenshot in the question).
I already got Segoe UI Emoji preinstalled though.

Edit:
Do we need to install any plugins such that Emojis can render properly?

![image](https://user-images.githubusercontent.com/57580593/150068603-93ed309b-e81b-4c35-bd41-2c8b256aff7c.png)

**Reply by Kethku** [reply]
No it should work by default. Try setting guifont to *. This will open a buffer with all the font names you should use.

Those emoji are coming from somewhere. I would also try (temporarily) using a default font like consolas as your first font to ensure the emoji aren't being taken from the Nerdfont you're using.

Lastly I would check to make sure that you are running a build from main.

**Reply by AnsonH** [reply]
After some investigation, the emojis work properly on [version 0.5.0](https://github.com/neovide/neovide/releases/tag/0.5.0). I was using [version 0.6.0](https://github.com/neovide/neovide/releases/tag/0.6.0) back then and the emojis are not rendering properly.

Thanks for the help!

**Reply by Kethku** [reply]
Please use a build from main. Many changes have gone in since the 0.6 release
